pkg/api/v1/schema.go

```diff
@@ -780,3 +781,8 @@ type PodNetwork struct {
 	// Default 10.0.2.0/24 if not specified.
 	VMNetworkCIDR string `json:"vmNetworkCIDR,omitempty"`
 }
+
+// Represents the multus cni network.
+// ---
+// +k8s:openapi-gen=true
+type MultusNetwork struct{}
```

Comment:
I guess you are using the `name` of the networkSource to map to the multus network. You need to do it instead like it is done for PVCs:

```yaml
  volumes:
    - name: myidentifier
      persistentVolumeClaim:
        claimName: myclaim
```

so:

```yaml
  networks:
   - name: mymultusconf
     multus:
       networkName: blue
```

The `name` is just used for expressing the correlation between an `interface` and a `network`, like it is used to express the relation between a `volumeMount` and a `volume` on Pods.

Reply:
@rmohr This will make a problem with this suggestion https://github.com/kubevirt/kubevirt/pull/1457

Reply:
I am actually suggesting to put something into the struct which is empty right now. How is that related to the issue? We have this kind of structure also on our disks.

Reply:
Sorry my bad the other PR is about the interface not the networks.
good comment I will make the change.

One question just like the volumes the mapping between the interface and the network will be base on the name object and not base on the networkName right?